test/api_tester.py

```diff
@@ -45,12 +45,12 @@ class ApiTester(unittest.TestCase):
 
     def setUp(self):
         self.failed_any_test = False
-        self.default_storage = 'Local-CI'
+        self.default_storage = 'Local-CI-Linux'
```

Comment:
Is it correct storage?

Reply:
Local debug info slipped into the commit. Fixed